bootstrap.js

```diff
@@ -5,6 +5,12 @@ module.exports = function(grunt) {
     config: config
   });
 
+  // Allow Gruntconfig.json to set default descriptions.
+  if (config.help) {
+    grunt.config('help', config.help);
+    grunt.config('config.help', '');
```

Comment:
@grayside, is this line necessary? I don't know of a use of `config.help` as a Grunt task.